langchain/retrievers/arxiv.py

```diff
@@ -0,0 +1,18 @@
+from typing import List
+
+from langchain.schema import BaseRetriever, Document
+from langchain.utilities.arxiv import ArxivAPIWrapper
+
+
+class ArxivRetriever(BaseRetriever, ArxivAPIWrapper):
```

Comment:
Would it make sense to change ArxivAPIWrapper into a DocumentLoader
and create a compositional pattern that allows upgrading a document loader into a retriever so we don't end up with double inheritance? 

@hwchase17 @leo-gan